packages/web-frontend/src/utils/getUniqueViewId.js

```diff
@@ -4,4 +4,7 @@
  */
 
 export const getUniqueViewId = ({ organisationUnitCode, dashboardGroupId, viewId }) =>
-  [organisationUnitCode, dashboardGroupId, viewId].join('_');
+  [organisationUnitCode, dashboardGroupId, viewId].join('___');
+
+export const getViewIdFromInfoViewKey = infoViewKey =>
+  infoViewKey ? infoViewKey.split('___')[2] || null : null;
```

Comment:
You may optionally prefer the following syntax:

```suggestion
export const getViewIdFromInfoViewKey = infoViewKey =>
  (infoViewKey && infoViewKey.split('___')[2]) || null;
```